features/steps/partcad-cli/commands/add/assembly.py

```
@@ -0,0 +1,36 @@
+from behave import given, when, then
```

Ruff (0.8.0)

`behave.when` imported but unused

Remove unused import: `behave.when`

(F401)